singer-io/tap-typeform
# Enable dev mode

# Description of change
- Dev mode implementation
- Unitests

# Related PRs

- Orchestrator [PR #299](https://github.com/stitchdata/orchestrator/pull/299)
- Connections-service [PR #1833](https://github.com/stitchdata/connections-service/pull/1833)

# Manual QA steps
1.  Verified when refresh token is not provided, access token is not refreshed 
2. Verified when refresh token and client details are provided, access token and refresh token is refreshed on discovery or at the start of sync.
3. Verified in dev mode, access token is not refreshed
4. Verified that in above scenarios, discovery and sync runs as expected

 
# Risks
 - 
 
# Rollback steps
 - revert this branch
